src/components/css/booklist.css

```diff
@@ -0,0 +1,173 @@
+* {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.container-all {
+  width: 90%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
```

Comment:
- You have done well so far, but your app is not styled the way it is in the design template. Please reference the image attached below. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project: 
- The spacing between each book is more than it is in the design template, please fix this 🙏
![thsthse](https://github.com/AliShahSafdari/bookstore/assets/73607512/2d46e054-cd00-4c4e-b038-9390f4995164)

Design template:
![eththe](https://github.com/AliShahSafdari/bookstore/assets/73607512/f5975738-ef94-4cbc-9ce7-2f26040f8a2a)

Reply:
I Fix it.